### ISSUE TITLE
bug: nixpkgs branchoff happened, version bump required

### ISSUE THREAD

**Issue by NobbZ** [nix-community/home-manager#4694]
### Are you following the right branch?

- [X] My Nixpkgs and Home Manager versions are in sync

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue description

```
You are using

  Home Manager version 23.11 and
  Nixpkgs version 24.05.

Using mismatched versions is likely to cause errors and unexpected
behavior. It is therefore highly recommended to use a release of Home
Manager that corresponds with your chosen release of Nixpkgs.

If you insist then you can disable this warning by adding

  home.enableNixpkgsReleaseCheck = false;

to your configuration.
```

Just reporting as the support requests in the Discord are coming in. I am not affected by myself as I am using standalone.

### System information

```markdown
------
```

**Comment by eclairevoyant** [comment]
will be fixed by #4686

**Comment by rycee** [comment]
Should be fixed now 🙂
